platform/ext/target/nordic_nrf/common/core/target_cfg.c

```diff
@@ -876,6 +876,16 @@ enum tfm_plat_err_t nvic_interrupt_target_state_cfg(void)
 #elif NRF_SECURE_UART_INSTANCE == 30
     /* UARTE30 is a secure peripheral, so its IRQ has to target S state */
     NVIC_ClearTargetState(NRFX_IRQ_NUMBER_GET(NRF_UARTE30));
+#elif NRF_SECURE_UART_INSTANCE == 00
+    NVIC_ClearTargetState(NRFX_IRQ_NUMBER_GET(NRF_UARTE00));
```

Comment:
Couldn't we simply do some macro concatenation here instead? Maybe something like (not sure how that works)
```suggestion
NVIC_ClearTargetState(NRFX_IRQ_NUMBER_GET(NRF_UARTE ##NRF_SECURE_UART_INSTANCE));
```